spec/requests/stories_index_spec.rb

```diff
@@ -231,20 +231,20 @@ def sets_nginx_headers
         create(:article, approved: false, body_markdown: u_body, score: 1)
       end
 
-      it "doesn't display posts with the campaign tags when sidebar is disabled" do
+      xit "doesn't display posts with the campaign tags when sidebar is disabled" do
```

Comment:
it's worth noting that the skipped tests here are _negative_ tests - specifying the absence of what was filtered out by the original strategy but is sometimes included in the weighted query strategy (experiment).